howso/engine/trainee.py

```diff
@@ -962,6 +967,76 @@ def auto_analyze(self) -> None:
             self.client.auto_analyze(self.id)
         else:
             raise ValueError("Client must have the 'auto_analyze' method.")
+    
+    def get_auto_ablate_params(self):
+        """
+        Get parameters set by :meth:`set_auto_ablate_params`.
```

Comment:
Can we update this docstring too?
The change to "Get auto-ablation parameters set by ..."